critiquebrainz/frontend/messages.pot

```diff
@@ -1848,7 +1848,7 @@ msgid "Can't find a review with the specified ID."
 msgstr ""
 
 #: critiquebrainz/frontend/views/review.py:75
-msgid "Review has been hidden. You need to be an administrator to view it."
+msgid "Review has been hidden. You need to be a moderator to view it."
```

Comment:
The `.mo` files are generated via the `pybabel compile` command. You may wanna just edit the code in a separate commit (the .py and the jinja files (if any)) and then use `pybabel` commands to generate the `messages.pot`(using `pybabel extract` command) and the `messages.po` (using `pybabel compile` command) files. Then add the changed files in a separate commit.